draft-gazdag-x509-hash-sigs.md

```diff
@@ -144,7 +167,33 @@ i.e. either 24, 32 or 64 bytes. The height of a single tree is typically given
 by the parameter 'h'. The number of levels of trees is either called 'L' (HSS)
 or 'd' (XMSS, XMSS^MT, SPHINCS+).
 
-# Subject Public Key Algorithms
+# Use Cases of HBS in X.509
+
+As many cryptographic algorithms that are considered to be quantum-resistant,
+HBS have several pros and cons regarding their practical usage. On the positive side
+they are considered to be secure against a classical as well as a quantum adversary,
+a secure instantiation of HBS may always be built as long as a cryptographically
+secure hash function exists. While key generation and signature generation may
+take longer than classical alternatives, fast and minimal verification routines
```

Comment:
Maybe add something like:
- In comparison to other post-quantum signature schemes stateful ones can have (small signature + public key) sizes.

Reply:
Nach dem Satz "... secure hash function exists." würde ich Folgendes einfügen:

Moreover, HBS offer small public key sizes, and, in comparison to other post-quantum signature schemes, the stateful HBS offer relatively small signauture (for certain parameter sets).